gustaf/spline/extract.py

```diff
@@ -141,7 +148,12 @@ def faces(spline, resolutions,):
         vertices = []
         faces = []
         offset = 0
-        kvs = spline.knot_vectors
+        # accomodate bezier Splines
+        if spline.whatami.startswith("Bezier"):
+            kvs = [[0., 1.]] * 3
+        else:
+            kvs = spline.knot_vectors
+
```

Comment:
I realized that this part also can be replaced by `unique_knots`. so maybe:
```
ukvs = spline.unique_knots
```
and replace `kvs` to `ukvs`